packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.js

```diff
@@ -107,10 +117,10 @@ export default class QuanticResultQuickview extends LightningElement {
 
   connectedCallback() {
     getHeadlessEnginePromise(this.engineId)
-      .then((engine) => {
+      .then((/** @type {import("coveo").SearchEngine<any>} */ engine) => {
         this.initialize(engine);
       })
-      .catch((error) => {
+      .catch((/** @type {{ message: any; }} */ error) => {
```

Comment:
Why are you adding this typings?  does it show you a red underline when you try to access `error.message` ?

Reply:
it showed a warning, but not an error, I can remove those :P